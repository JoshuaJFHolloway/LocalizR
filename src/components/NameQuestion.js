import React from 'react'

const NameQuestion = () => {
  return (
    <h3> What is your name? </h3>
  )
};

export default NameQuestion;