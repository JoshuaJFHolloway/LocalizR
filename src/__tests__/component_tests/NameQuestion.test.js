import React from 'react';
import { shallow, mount } from 'enzyme';
import NameQuestion from '../../components/NameQuestion';

describe('NameQuestion', () => {
  let namequestion = shallow(<NameQuestion />);

  it('renders correctly', () => {
    expect(namequestion).toMatchSnapshot();
  });

});