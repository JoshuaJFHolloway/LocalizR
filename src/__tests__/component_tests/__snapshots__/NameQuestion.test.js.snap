// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NameQuestion renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NameQuestion />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": " What is your name? ",
    },
    "ref": null,
    "rendered": " What is your name? ",
    "type": "h3",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": " What is your name? ",
      },
      "ref": null,
      "rendered": " What is your name? ",
      "type": "h3",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
